src/displayapp/screens/WatchFacePrideFlag.cpp

```diff
@@ -0,0 +1,376 @@
+#include "displayapp/screens/WatchFacePrideFlag.h"
+#include <lvgl/lvgl.h>
+#include "displayapp/screens/Symbols.h"
```

Comment:
includes removed from the header are missing

```suggestion
#include "components/battery/BatteryController.h"
#include "components/ble/BleController.h"
#include "displayapp/screens/Symbols.h"
```